plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/before/test/impl/CloseWelcomeScreenExt.java

```diff
@@ -73,4 +73,9 @@ public boolean hasToRun() {
 		return CloseWelcomeScreenExt.CLOSE_WELCOME_SCREEN;
 	}
 
+	@Override
+	public long getPriority() {
+		return 1000000;
```

Comment:
All constants should be be defined as constants (public static final). Also update the JavaDoc so that it is clear what's the priority.